django_sorcery/db/models.py

```diff
@@ -423,3 +427,68 @@ def full_clean_flush_handler(session, **kwargs):
     for i in session.dirty | session.new:
         if isinstance(i, Base):
             i.full_clean()
+
+
+def _coerce(target, value, oldvalue, initiator, form_field):
+    value = value.strip() if isinstance(value, six.string_types) else value
```

Comment:
hmm. maybe honor `CharField.strip`? https://docs.djangoproject.com/en/2.1/ref/forms/fields/#django.forms.CharField

Reply:
what about junk values passed to decimal fields? see the tests

Reply:
also i don't think `strip` is being used during field mapping